templates/taxi/car_form.html

```diff
@@ -0,0 +1,10 @@
+{% extends 'base.html' %}
+{% load crispy_forms_filters %}
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} Car</h1>
+  <form action="" method="post">
+    {% csrf_token %}
+    {{ form|crispy }}
+    <input class="btn btn-primary" type="submit" value="Submit">
+  </form>
```

Comment:
Consider adding a Cancel button to allow users to navigate back without submitting changes. This is important for user experience and consistency across forms.